task/deprecated-image-check/0.4/deprecated-image-check.yaml

```diff
@@ -53,20 +55,43 @@ spec:
         trap 'handle_error $(results.TEST_OUTPUT.path)' EXIT
 
         IMAGES_TO_BE_PROCESSED_PATH="/tmp/images_to_be_processed.txt"
-        SBOM_FILE_PATH="/tmp/sbom.json"
+        touch /tmp/images_to_be_processed.txt
 
         success_counter=0
         failure_counter=0
         error_counter=0
         warnings_counter=0
 
-        # Get base images from SBOM
-        cosign download sbom "${IMAGE_URL}@${IMAGE_DIGEST}" > ${SBOM_FILE_PATH}
+        images_processed_template='{"image": {"pullspec": "'"$IMAGE_URL"'", "digests": [%s]}}'
+        digests_processed=()
+
+        imagewithouttag=$(echo -n $IMAGE_URL | sed "s/\(.*\):.*/\1/")
+        # strip new-line escape symbol from parameter and save it to variable
+        imageanddigest=$(echo $imagewithouttag@$IMAGE_DIGEST)
```

Comment:
Suggestion to use `echo -n` just in case here.

Reply:
Done